modules/algebrasAndModel/src/main/scala/utils/Types.scala

```diff
@@ -25,11 +25,19 @@ object Types {
       slackUserId: SlackUserId,
       slackChannelId: SlackChannelId
   )
-  final case class Organization(organization: String) extends AnyVal
-  object Organization {
-    implicit val organizationEncoder: Encoder[Organization] = deriveUnwrappedEncoder[Organization]
-    implicit val organizationDecoder: Decoder[Organization] = deriveUnwrappedDecoder[Organization]
+
+  final case class Owner(owner: String) extends AnyVal
+  object Owner {
+    implicit val encoder: Encoder[Owner] = deriveUnwrappedEncoder[Owner]
+    implicit val decoder: Decoder[Owner] = deriveUnwrappedDecoder[Owner]
```

Comment:
Nitpick: you could also either use circe Codec (`trait Codec[A] extends Decoder[A] with Encoder[A]`) to make instance definition more concise.

*Below is not necessary for you to do, it's just FYR*

If you want to go the extra mile, you could use @newtype macro instead of value classes and  derevo @derive macro annotation:

```scala
@derive(encoder, decoder)
@newtype
case class Owner(owner: String)
```